src/components/molecules/Settings/SettingPage/index.tsx

```diff
@@ -35,7 +46,10 @@ const SettingPage: React.FC<Props> = ({
           currentProject && <ProjectMenu currentProject={currentProject} teamId={currentTeam?.id} />
         }
       />
-      <BodyWrapper>
+      <BodyWrapper
+        onScroll={e => {
+          !loading && hasMoreItems && handleScroll(e, onScroll);
+        }}>
```

Comment:
I believe this is why the project list isn't scrolling. You have the scroll on the body wrapper, which I don't think is able to be scrolled bc the height is 100%.
If you do this, you need to set the height here to be a little more than 100vh, OR leave it 100% here and on this component's parent set it to like 105vh or something.

Reply:
should be fixed now